src/cartridges/int_adyen_SFRA/cartridge/client/default/js/adyen_checkout/checkoutConfiguration.js

```diff
@@ -267,9 +267,15 @@ function getGiftCardConfig() {
 
 function handleOnChange(state) {
   let { type } = state.data.paymentMethod;
+  const multipleTxVariantsMethods = constants.MULTIPLE_TX_VARIANT_METHODS;
+  // Bug in the UPI component, as soon as its fixed this line of code is not needed anymore
+  store.componentsObj.upi.isValid = false;
   if (store.selectedMethod === 'googlepay' && type === 'paywithgoogle') {
     type = 'googlepay';
   }
+  if (multipleTxVariantsMethods.includes(store.selectedMethod)) {
```

Comment:
we can possibly refactor the google pay if statement into this multipleTxVariantsMethods. But can also be done at a different PR